### DISCUSSION TITLE
Issue with running a model on mac

### DISCUSSION THREAD

**Discussion by mohsen-ghaffari1992** [UPPAALModelChecker/UPPAAL-Meta#205] [Q&A]
Hi,

Recently, I attempted to open a system that was previously tested by my colleagues. Strangely, they encountered no difficulties, whereas I keep encountering an error. To provide you with a better understanding, I have attached some screenshots illustrating the issue.
<img width="553" alt="Screenshot 2023-06-12 at 15 11 59" src="https://github.com/UPPAALModelChecker/UPPAAL-Meta/assets/77638469/59c5555a-e798-480f-b3b5-ca1abf67b264">
<img width="620" alt="Screenshot 2023-06-12 at 15 12 20" src="https://github.com/UPPAALModelChecker/UPPAAL-Meta/assets/77638469/18232694-1637-4163-b969-b7426ed67a71">
<img width="1047" alt="Screenshot 2023-06-12 at 15 12 34" src="https://github.com/UPPAALModelChecker/UPPAAL-Meta/assets/77638469/2a5f5cfe-2ee8-4ac6-a86f-a56fa8372159">

I am using uppaal64-4.1.26, and system is only an xml file (no extra libraries).

Thank you in advance for your valuable time and ongoing support.

**Comment by mikucionisaau** [comment]
I made this minimal model:
```c
clock x;
const double Capacity = 2.5;
double value(){
  if (x<=Capacity)
    return 1.5;
  else
    return 2.5;
}
```
4.1.26 does not allow it (with "Boolean expected" error) as it does not implement the necessary support for floating point variables.

This model works fine in Uppaal Stratego-11 and Uppaal 5 rc4.

**Comment by mohsen-ghaffari1992** [comment]
Thanks Marius!

I moved to Uppaal Stratego and everything is fine now.

Best,
Mohsen

From: Marius Mikučionis ***@***.***>
Date: Thursday, 15 June 2023 at 08.09
To: UPPAALModelChecker/UPPAAL-Meta ***@***.***>
Cc: Mohsen Ghaffari ***@***.***>, Author ***@***.***>
Subject: Re: [UPPAALModelChecker/UPPAAL-Meta] Issue with running a model on mac (Discussion #205)

I made this minimal model:

clock x;

const double Capacity = 2.5;

double value(){

  if (x<=Capacity)

    return 1.5;

  else

    return 2.5;

}

4.1.26 does not allow it (with "Boolean expected" error) as it does not implement the necessary support for floating point variables.

This model works fine in Uppaal Stratego-11 and Uppaal 5 rc4.
